teps/0096-pipelines-v1-api.md

```diff
@@ -174,11 +182,106 @@ V1 work should prioritize stabilizing existing beta features that support the [u
+- Ensure metrics are accurate and mark all metrics as "alpha" or "stable". 
+
+### Stability
+
+- Track bug fixes and small backwards incompatible changes outside of this TEP (e.g. in a Github project or milestones).
+- Implement conversion webhook for v1beta1 objects.
+
+#### CRD Stability Levels
+
+- The Task, TaskRun, Pipeline, and PipelineRun CRDs should be moved to the "stable" stability level.
+- Because Custom Task is mission critical to our extensibility use case, Run should be upgraded to beta stability.
+  - See [Custom Tasks Beta](https://github.com/tektoncd/pipeline/issues/4313) for more information on work required to upgrade Custom Tasks to beta.
+- Keep the ClusterTask CRD at beta stability.
+  - ClusterTask functionality will likely be replaced by [remote resolution](./0060-remote-resource-resolution.md), but there's no plan
+  to replace this functionality at a beta level of stability before releasing a V1 API.
+  - [Still under discussion]: We will decide before v1 whether to stabilize or remove ClusterTask pending community feedback on ClusterTask and remote resolution.
+  See [Where should we take ClusterTasks next?](https://github.com/tektoncd/pipeline/issues/4476) for more info.
+
+| CRD         | Current level | Proposed level |
+| ----------- | ------------- | -------------- |
+| Task        | beta          | stable         |
+| TaskRun     | beta          | stable         |
+| Pipeline    | beta          | stable         |
+| PipelineRun | beta          | stable         |
+| ClusterTask | beta          | beta           |
+| Run         | alpha         | beta           |
+
+#### API Changes
+
+- Fix [pain points](https://github.com/tektoncd/pipeline/issues/3792) associated with TaskRun and PipelineRun Status.
+- Consider replacing any Task or Pipeline fields that should support parameterization with Strings and performing our own validation.
+  - See [Handling parameter interpolation in fields not designed for it](https://github.com/tektoncd/pipeline/issues/1530) for more details.
+
+#### Deprecations
+
+- Deprecate `PipelineRun.Timeout` and [upgrade `PipelineRun.Timeouts` to beta](https://github.com/tektoncd/pipeline/issues/4460).
+- Remove `Pipeline.Tasks[].Conditions`. Conditions have been replaced by when expressions, which are already in beta.
+- Because PipelineResources are deprecated, remove the `Resources` fields of `Task`, `TaskRun`, `Pipeline`, and `PipelineRun`.
+    - [Still under discussion]: Additional community feedback is needed to determine the minimum V1 requirements that will make it usable for PipelineResource users. We should provide PipelineResource users with a migration path to the V1 API, which may include blocking on features proposed in [TEP-0074: Deprecate PipelineResources](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md).
```

Comment:
TEP-0044 is a very large and somewhat contentious topic :( 

I agree that users input would be very valuable - thanks for starting the mailing list thread about this.
I hope we're able to identify a minimalistic set of things we need to do for `v1` and cover any extra requirements afterwards.